mappings/net/minecraft/client/render/block/entity/BannerBlockEntityRenderer.mapping

```diff
@@ -1,7 +1,20 @@
 CLASS net/minecraft/class_823 net/minecraft/client/render/block/entity/BannerBlockEntityRenderer
-	FIELD field_20810 field Lnet/minecraft/class_630;
-	FIELD field_20811 verticalBar Lnet/minecraft/class_630;
-	FIELD field_20812 topBar Lnet/minecraft/class_630;
+	FIELD field_20810 banner Lnet/minecraft/class_630;
+	FIELD field_20811 pillar Lnet/minecraft/class_630;
+	FIELD field_20812 crossbar Lnet/minecraft/class_630;
 	METHOD <init> (Lnet/minecraft/class_824;)V
 		ARG 1 dispatcher
-	METHOD method_24080 createField ()Lnet/minecraft/class_630;
+	METHOD method_23802 render (Lnet/minecraft/class_2573;Lnet/minecraft/class_4587;Lnet/minecraft/class_4597;IILnet/minecraft/class_630;Lnet/minecraft/class_4730;Z)V
+		ARG 0 entity
+		ARG 1 matrices
+		ARG 2 provider
+		ARG 3 light
+		ARG 4 overlay
+		ARG 5 part
+		ARG 6 texture
+		ARG 7 isBanner
```

Comment:
Really? Does this mean if the banner is in inventory?

Reply:
If it's a banner/banner or banner/on a shield.

Reply:
Or rather banner/on a stick or banner on a shield.

It's called by the shield renderer with a value of false and called with true by everything else.

Reply:
Since, as you mention, a banner is just the cloth (whether it's attached to a stick or a shield), this should be named `isBannerBlock` or something similar.